cvat/apps/engine/serializers.py

```diff
@@ -3499,6 +3505,51 @@ class AssetWriteSerializer(WriteOnceMixin, serializers.ModelSerializer):
     filename = serializers.CharField(required=True, max_length=MAX_FILENAME_LENGTH)
     guide_id = serializers.IntegerField(required=True)
 
+    @staticmethod
+    def write_asset(
```

Comment:
This method seems like it should be a custom `save` implementation.

Reply:
Yes, reimplemented